src/providers/grass/CMakeLists.txt

```diff
@@ -186,6 +186,7 @@ MACRO(ADD_GRASSLIB GRASS_BUILD_VERSION)
       TARGET_LINK_LIBRARIES(grassprovider${GRASS_BUILD_VERSION}
         qgis_gui
       )
+      ADD_DEPENDENCIES(grassprovider ui)
```

Comment:
```suggestion

```

Reply:
Sorry, it should be

```suggestion
      ADD_DEPENDENCIES(grassprovider${GRASS_BUILD_VERSION} ui)
```

Can you confirm?

Reply:
I confirm that it works with this last fix. LGTM